js/foam/browser/u2/BrowserView.js

```diff
@@ -19,431 +19,357 @@ CLASS({
+        top: 0;
+        bottom: 0;
+        left: 0;
+        right: 0;
+        z-index: 20;
+        transition: visibility 0.01s linear <%= ANIMATION_TIME %>;
+      }
+      $-menu-container$-menu-open {
+        visibility: visible;
+        transition: visibility <%= ANIMATION_TIME %> linear;
+      }
 
-          $-menu-body {
-            background-color: #fff;
-            position: absolute;
-            height: 100%;
-            top: 0;
-            left: 0;
-            transform: translate3d(-300px, 0, 0);
-            overflow-y: auto;
-            width: 300px;
-            transition: transform <%= ANIMATION_TIME %> ease;
-          }
-          $-menu-open $-menu-body {
-            transform: translate3d(0, 0, 0);
-            transition: transform <%= ANIMATION_TIME %> ease;
-          }
+      $-menu-inner {
+        position: relative;
+        height: 100%;
+        width: 100%;
+      }
+      $-menu-overlay {
+        background-color: #000;
+        opacity: 0;
+        position: absolute;
+        height: 100%;
+        top: 0;
+        left: 0;
+        width: 100%;
+        transition: opacity <%= ANIMATION_TIME %> ease;
+      }
+      $-menu-open $-menu-overlay {
+        opacity: 0.4;
+        transition: opacity <%= ANIMATION_TIME %> ease;
+      }
 
-          $-body {
-            display: flex;
-            flex-direction: column;
-            flex-grow: 1;
-            overflow-x: hidden;
-            overflow-y: auto;
-          }
-        */},
-      ],
+      $-menu-body {
+        background-color: #fff;
+        position: absolute;
+        height: 100%;
+        top: 0;
+        left: 0;
+        transform: translate3d(-300px, 0, 0);
+        overflow-y: auto;
+        width: 300px;
+        transition: transform <%= ANIMATION_TIME %> ease;
+      }
+      $-menu-open $-menu-body {
+        transform: translate3d(0, 0, 0);
+        transition: transform <%= ANIMATION_TIME %> ease;
+      }
 
-      methods: [
-        function initE() {
-          var self = this;
-          this.cls(this.myCls());
-          this.start('style')
-              .add('.' + this.myCls('header-color') + ' { background-color: ' +
-                  this.data.headerColor + '; }')
-              .end();
-          var header = this.start('div')
-              .x({ data: this })
-              .cls(this.myCls('header'))
-              .cls(this.myCls('header-color'))
-              .enableCls('foam-u2-Element-hidden', this.searchMode$);
+      $-body {
+        display: flex;
+        flex-direction: column;
+        flex-grow: 1;
+        overflow-x: hidden;
+        overflow-y: auto;
+      }
+    */},
+  ],
 
-          header.add(this.MENU_BUTTON);
-          header.start('span')
-              .cls(this.myCls('header-title'))
-              .add(this.title$)
-              .end();
+  methods: [
+    function initE() {
+      var self = this;
+      this.cls(this.myCls());
+      this.start('style')
```

Comment:
It was intended to make the browser's header colour configurable without having to add CSS overrides to the right class names. I don't think it really works anymore anyway.

Why is this so busted for security? This isn't user data, it's programmer-controlled. We can escape it too if we need to. We could just turn this into `.style()` calls on elements that add toolbars, rather than this pseudoclass. Should I do that?